src/NuGet/BuildNuGets.csx

```diff
@@ -281,33 +281,17 @@ int PackFiles(string[] nuspecFiles, string licenseUrl)
         }
 
         p.StartInfo.UseShellExecute = false;
-        p.StartInfo.RedirectStandardError = true;
 
-        Console.WriteLine($"{Environment.NewLine}Running: nuget pack {file} {commonArgs}");
+        Console.WriteLine($"Packing {file}");
 
         p.Start();
         p.WaitForExit();
 
         var currentExit = p.ExitCode;
         if (currentExit != 0)
         {
-            var stdErr = p.StandardError.ReadToEnd();
-            string message;
-            if (BuildingReleaseNugets && stdErr.Contains("A stable release of a package should not have a prerelease dependency."))
-            {
-                // If we are building release nugets and if any packages have dependencies on prerelease packages
-                // then we want to ignore the error and allow the build to succeed.
-                currentExit = 0;
-                message = $"{file}: {stdErr}";
-                Console.WriteLine(message);
-            }
-            else
-            {
-                message = $"{file}: error: {stdErr}";
-                ReportError(message);
-            }
-
-            File.AppendAllText(ErrorLogFile, Environment.NewLine + message);
+            Console.WriteLine($"nuget pack {p.StartInfo.Arguments}");
+            ReportError($"Pack operation failed with {currentExit}");
```

Comment:
@jasonmalinowski removed all of the standard output / error redirection here. Turns out it was being done to not fail the script when we got the very specific error around building release packages with pre-release dependencies. That is actually no longer an error in Nuget 4.0 hence this is unnecessary